test/controllers/tasks_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```

Comment:
since you are using `rspec`, why do you still write your test in the `test` folder?